shared/video-sdk/encrypt-media-streams/project-test/react-native.mdx

```diff
@@ -0,0 +1,10 @@
+- *Android*
+
+    1. Enable the Developer options on your Android device, and then connect it to your computer using a USB cable.
+    1. Run npx react-native run-android in the project root directory.
+
+- *iOS:*
+
+    1. Open the ProjectName/ios/ProjectName.xcworkspace folder with Xcode.
+    1. Connect your iOS device to your Mac using a USB cable.
+    1. Click the Build and run button in Xcode.
```

Comment:
In Xcode, run your project. 


They are developers, they should have figured this one out :-).